policyengine_us/variables/gov/usda/snap/income/deductions/self_employment/snap_self_employment_expense_deduction.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class snap_self_employment_expense_deduction(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = (
+        "Self-employment income deduction for calculating SNAP benefit amount"
+    )
+    label = "SNAP self-employment expense deduction"
+    unit = USD
+    reference = "https://www.snapscreener.com/blog/self-employment"
+
+    def formula(spm_unit, period, parameters):
+        self_employment_income = add(
+            spm_unit, period, ["self_employment_income"]
+        )
+        expenses = spm_unit("snap_self_employment_income_expense", period)
+        p = parameters(period).gov.usda.snap.income.deductions.self_employment
+        state_code = spm_unit.household("state_code", period)
+        income_percentage = max_(
```

Comment:
`x_percentage` should be a %
```suggestion
        income_percentage = p.rate[state_code]
        percentage_based_amount = max_(income_percentage * self_employment_income, 0)
```